index.html

```diff
@@ -13,33 +13,252 @@
+          <section class="img-placeholder">
+            <section class="div">
+              <section class="right-works">
+                <h2 class="pro">Profesional Art Printing Data</h2>
+                <h6 class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6>
+                <ul class="tags">
+                  <li class="tags1"><h6>html</h6></li>                                            
+                  <li class="tags2"><h6>bootstrap</h6></li>
+                  <li class="tags3"><h6>Ruby</h6></li>
+                </ul>
+              </section>
+            </section>
+          </section>
+        </section>
+        <section class="action">
+          <section class="but">
+            <a href="#" class="see1">See Project</a>
+          </section>
+        </section>
+      </section>
+      <section class="card-works">
+        <section class="mask">
+          <section class="img-placeholder">
+            <section class="div">
+              <section class="right-works">
+                <h2 class="pro">Profesional Art Printing Data</h2>
+                <h6 class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6>
+                <ul class="tags">   
+                  <li class="tags1"><h6>html</h6></li>                                            
+                  <li class="tags2"><h6>bootstrap</h6></li>
+                  <li class="tags3"><h6>Ruby</h6></li>
+                </ul>
+              </section>
+            </section>
+          </section>
+        </section>
+        <section class="action">
+          <section class="but">
+            <a href="#" class="see1">See Project</a>
+          </section>
+        </section>
+      </section>
+      <section class="card-works">
+        <section class="mask">
+          <section class="img-placeholder">
+            <section class="div">
+              <section class="right-works">
+                <h2 class="pro">Profesional Art Printing Data</h2>
+                <h6 class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6>
+                <ul class="tags">
+                  <li class="tags1"><h6>html</h6></li>                                            
+                  <li class="tags2"><h6>bootstrap</h6></li>
+                  <li class="tags3"><h6>Ruby</h6></li>
+                </ul>
+              </section>
+            </section>
+          </section>
+                        <!-- <section class="mask-sub"></section> -->
+        </section>
+        <section class="action">
+          <section class="but">
+            <a href="#" class="see1">See Project</a>
+          </section>
+        </section>
+      </section>
+    </section>
+    <section class="about">
+      <h1 class="primary">About Me</h1>
```

Comment:
[Optional]
- [ ] A web page should have only one `h1` tag for semantic reasons the rest of the subheaders should be `h2`, `h3`... It's better if this subheader is an `h2` instead of an `h1`

Reply:
Okay Thank you very much